### PR TITLE
[tests][s]: qa tests for data-get command - refs #299

### PR DESCRIPTION
**Pull request by Mikanebu**
* data get QA tests
  * excel file remote
  * small dataset from DataHub
  * medium dataset from DataHub
  * big dataset from DataHub

### CODE REVIEW

**Review by zelima — APPROVED** [review]
Looks fine

**Review by anuveyatsu — CHANGES_REQUESTED** [review]
Looks OK, but I think we should implement some function to delete all temporary folders and files in it. Found this nice recursive function to do it: https://stackoverflow.com/a/32197381/6502239

File: test/cli.test.js [hunk 1]
```diff
@@ -37,44 +37,27 @@ const runcli = (...args) => {
...
+          deleteFolderRecursive(curPath);
+        } else { // delete file
+          fs.unlinkSync(curPath);
+        }
+      })
+      fs.rmdirSync(path);
+    }
+  }
+  deleteFolderRecursive('finance-vix')
+  deleteFolderRecursive('test/small-dataset-100kb')
```
Comment (anuveyatsu) [inline test/cli.test.js:55]: @Mikanebu wouldn't it work if you do `deleteFolderRecursive('test')`?

Comment (Mikanebu) [reply test/cli.test.js:55]: @anuveyatsu then, it will delete `fixtures` directory and all other files including `test` directory itself

Comment (anuveyatsu) [reply test/cli.test.js:55]: @Mikanebu aha, the problem is that our username is `test` and we already have that dir

**Review by anuveyatsu — APPROVED** [review]
Looks good

**Pull request merged**